ng/src/web/components/dialog/dialog.js

```diff
@@ -22,15 +23,31 @@
  */
 
 import React from 'react';
+import glamorous from 'glamorous';
 
-import {KeyCode, has_value} from 'gmp/utils.js';
+import {KeyCode, has_value, is_defined} from 'gmp/utils.js';
 
 import PropTypes from '../../utils/proptypes.js';
 
 import DialogContainer from './container.js';
 import DialogOverlay from './overlay.js';
 
 const DEFAULT_DIALOG_WIDTH = '800px';
+const DEFAULT_DIALOG_HEIGHT = '550px';
+const DEFAULT_DIALOG_MIN_HEIGHT = 250;
+const DEFAULT_DIALOG_MIN_WIDTH = 450;
+
+const Resizer = glamorous.div({
+  position: 'absolute',
+  bottom: '.4em',
```

Comment:
The resizer is used at dialog.js and withDialog.js so it should be put into an own js module (dialog/resizer.js) and imported in both.